src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final int DIVIDER_FOR_AREA = 2;
+    private int upperSide;
+    private int lowerSide;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int upbothside, int downbothside, int height) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. For example, before calling the constructor, calculate the values for 'upperSide', 'lowerSide', and 'height', then pass those variables to the constructor.